_data/companies.yml

```diff
@@ -277,6 +277,13 @@
   visitors: Restricted 
   events: Restricted 
   last_update: 2020-03-05 
+  
+- name: DigitalOcean
+  wfh: Encouraged
+  travel: Restricted
+  visitors: Restricted
+  events: Restricted
+  last_update: 2020-03-02
```

Comment:
Is that a typo?
```suggestion
  last_update: 2020-03-12
```

Reply:
depends on which update you're interested in. 03-02 is when then restrictions have been shared with the team, 03-12 is when this entry has been last updated. ;)